### PR TITLE
Add difficulty indicator

### PR DESCRIPTION
**Pull request by andyg0808**
Basic indicator of the current difficulty score on an item, or "?" if there's none set:
image

### CODE REVIEW

**Comment by andyg0808** [comment]
deploy_block :+1: on #12

**Comment by andyg0808** [comment]
Connects #7

**Comment by danielbeardsley** [comment]
Rebased onto base pull after merging master and fixing it.

**Comment by copperwall** [comment]
This looks good to me. CR :fire:

**Comment by danielbeardsley** [comment]
Rebased (no conflicts) this to update the commit list and diff.

**Comment by copperwall** [comment]
CR :fire:

File: controllers/githubHooks.js [hunk 1]
```diff
@@ -76,8 +76,11 @@ var HooksController = {
 
          // Update DB with new pull request content.
          dbUpdated = preUpdate.then(function() {
-            return dbManager.updatePull(new Pull(body.pull_request));
-         });
+            var pull = new Pull(body.pull_request);
+            // Update the pull with information from the related issue
+            return pull.syncToIssue();
```
Comment (danielbeardsley) [inline controllers/githubHooks.js]: I think this logic may be reversed. The name is a bit hard to parse, and perhaps that's why. This function loads the *issue* from the DB and updates the *pull request* with issue data from the DB. To me, it seems like we need to do that when github tells us the issue is updated, not when github tells us the pull is.

**Comment by lexahall** [comment]
dev_block :hand: on merge conflicts

**Comment by entendu** [comment]
I _think_ I resolved the merge conflict properly, please take a look

**Comment by danielbeardsley** [comment]
We typically don't have scores on pulls... we'd need a tool to copy them over or something...

**Comment by lexahall** [comment]
un_dev_block 💯

**Comment by copperwall** [comment]
If story points are now more of an issue thing than a pull request thing, is it worth adding info about it on the page?

**Comment by colebemis** [comment]
I agree with @copperwall. Aren't story points more useful for milestone planning, not CR/QA?

**Comment by BaseInfinity** [comment]
CR 👙

**Comment by andyg0808** [comment]
dev_block :-1: on deciding if we want this.

**Comment by andyg0808** [comment]
It looks like it might not be that hard to support grabbing the size score from the issue when the pull first shows up (at least for pulls that only close one issue). But when the size on an issue is updated, it will likely require a manual refresh on each related pull to show up on them, because the issue doesn't necessarily know which pulls depend on it.
